src/main/resources/assets/inventorytweaks/ItemTree.xml

```diff
@@ -430,6 +445,7 @@
                     <diamondBlock id="diamond_block"/>
                     <emeraldBlock id="emerald_block"/>
                     <quartzBlock id="quartz_block"/>
+					<purpurPillar id="purpur_pillar"/>
```

Comment:
I was having trouble placing the pillar block. I placed with the Quartz block since Quartz also a pillar from but it is based off DV.